core/lib/types/src/l2/mod.rs

```diff
@@ -299,7 +299,7 @@ fn signature_to_vrs(signature: &[u8], tx_type: u32) -> (Option<U64>, Option<U256
 
     if let Ok(sig) = signature {
         (
-            Some(U64::from(sig.v())),
+            Some(U256::from(sig.v())),
```

Comment:
why? It's U64 in ethereum

Reply:
because the idea is to change everything to U256 in the server for ChainId. If chainId is U256 then v will also be U256